src/main/java/org/tron/core/capsule/StorageRowCapsule.java

```diff
@@ -26,7 +26,7 @@
 
 @Slf4j
 public class StorageRowCapsule implements ProtoCapsule<byte[]> {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Move this left curly brace to the beginning of next line of code. ([squid:LeftCurlyBraceStartLineCheck](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1664&issues=AWbJU6rASgN4uCclo1g_&open=AWbJU6rASgN4uCclo1g_)

[ISSUE_KEY]: AWbJU6rASgN4uCclo1g_